source/shared/cpp/ObjectModel/Enums.h

```diff
@@ -402,12 +402,22 @@ namespace AdaptiveSharedNamespace
 
     enum class ContainerBleedDirection
```

Comment:
Please add a comment here explaining the issues with java and/or swig which require us to explicitly define all of these combinations.